src/tests/option_round/options_sold_tests.cairo

```diff
@@ -105,65 +111,54 @@ fn test_total_options_after_auction_2() {
-    let params: OptionRoundParams = round_facade.get_params();
-    // OB 1 and 2 bid for > the total options available at the reserve price
-    let bid_count = 2;
-    let bid_price = params.reserve_price;
-    let bid_amount = bid_count * bid_price;
 
-    round_facade.place_bid(bid_amount, bid_price, option_bidder_buyer_1());
-    // Settle auction
-    vault_facade.timeskip_and_end_auction();
+    // Deposit liquidity and start the auction
+    accelerate_to_auctioning(ref vault_facade);
+    // Make bids
+    let mut current_round_facade: OptionRoundFacade = vault_facade.get_current_round();
+    let _params: OptionRoundParams = current_round_facade.get_params();
 
-    // Check total options sold is successful bids count
-    assert(bid_count == round_facade.total_options_sold(), 'options sold wrong');
+    // place bid and end the auction
+    accelerate_to_running(ref vault_facade);
+    let bid_count = 2;
+    assert(bid_count == current_round_facade.total_options_sold(), 'options sold wrong');
 }
 
 #[test]
 #[available_gas(10000000)]
 fn test_total_options_after_auction_4() {
     let (mut vault_facade, _) = setup_facade();
-    // LP deposits (into round 1)
-    let deposit_amount_wei: u256 = 10000 * decimals();
-    vault_facade.deposit(deposit_amount_wei, liquidity_provider_1());
-    // Start auction
-    vault_facade.start_auction();
-    let mut round_facade: OptionRoundFacade = vault_facade.get_current_round();
+    // Deposit liquidity and start the auction
+    accelerate_to_auctioning(ref vault_facade);
+    // Make bids
+    let mut current_round_facade: OptionRoundFacade = vault_facade.get_current_round();
+    let _params: OptionRoundParams = current_round_facade.get_params();
+
     // Make no bids
     // Settle auction
     vault_facade.timeskip_and_end_auction();
+
     // Check no options were sold if no bids
-    assert(0 == round_facade.total_options_sold(), 'no options should sell');
+    assert(0 == current_round_facade.total_options_sold(), 'no options should sell');
 }
 
 #[test]
 #[available_gas(10000000)]
 fn test_total_options_after_auction_5() {
     let (mut vault_facade, _) = setup_facade();
-    // LP deposits (into round 1)
-    let deposit_amount_wei: u256 = 10000 * decimals();
-    vault_facade.deposit(deposit_amount_wei, liquidity_provider_1());
-    // Start auction
-    vault_facade.start_auction();
-    let mut round_facade: OptionRoundFacade = vault_facade.get_current_round();
-    let params: OptionRoundParams = round_facade.get_params();
-    // Make bids for > total options
-    let bid_count = params.total_options_available + 10;
-    let bid_price = params.reserve_price;
-    let bid_amount = bid_count * bid_price;
 
-    round_facade.place_bid(bid_amount, bid_price, option_bidder_buyer_1());
-    // Settle auction
-    vault_facade.timeskip_and_end_auction();
+    // Deposit liquidity and start the auction
+    accelerate_to_auctioning(ref vault_facade);
```

Comment:
should use custom accelerator with previous values. This test is for bidding > total options available @ reserve price